Transforms/CachedQvvs/Editor/QvvsSceneViewFocus.cs

```diff
@@ -0,0 +1,41 @@
+using Latios.Transforms;
+using Unity.Entities.Editor;
+using Unity.Rendering;
+using UnityEditor;
+using UnityEngine;
+
+static class QvvsSceneViewFocus
+{
+    [UnityEditor.Callbacks.DidReloadScripts]
+    private static void ScriptsHasBeenReloaded()
+    {
+        SceneView.duringSceneGui += DuringSceneGui;
+    }
+
+    private static void DuringSceneGui(SceneView sceneView)
+    {
+        var ev = Event.current;
+
+        if (ev.type != EventType.ExecuteCommand || ev.commandName != "FrameSelected") return;
+
+        // Matches a runtime entity with no baked gameobject in hierarchy
+        if (Selection.objects.Length != 1 || 
+            Selection.objects[0] is not EntitySelectionProxy selectionProxy ||
+            Selection.activeGameObject != null) return;
+            
+        var entity = selectionProxy.Entity;
+        var w      = selectionProxy.World;
+        var em     = w.EntityManager;
```

Comment:
Nit: Just `var em = selectionProxy.World.EntityManager` is sufficient.

Reply:
Actually, now that I think about it, we should probably check that the `World` is a `LatiosWorld`, and if not, `return` here to preserve "inert behavior until a bootstrap is installed".